### DISCUSSION TITLE
Sound settings edited

### DISCUSSION THREAD

**Discussion by iconic2** [rafaelmardojai/blanket#250] [Q&A]
Hi there,
I have a question about blanket and how it might impact sound settings. 
I've been using it for a few days now and since i've downloaded blanket, im having problems with other app's like spotify, or just sounds from my browser (firefox). Everytime I open these apps, my sound automatically comes from my monitor, even though it is set to my headset in the settings. 
To change this, I have to manually change the sound to my second monitor, and then to my headset to get sound throug my headset. 
The weird thing however, is that blanket still plays through my headset, even if all other sound play through my monitor speakers. 
I don't launch blanket on startup, so thats why i'm hesitant if it is caused by the app, but I still wanted to ask, in case this is a known issue or in case you guys know a fix.
My os is Ubuntu 20.04.4 

Thanks in advance!

**Comment by rafaelmardojai** [comment]
There's no reason for this to happen. Blanket doesn't manipulate sound settings, it just plays audio in a standard way using the GStreamer framework.

Probably there's a problem in your system that happens to be triggered by Blanket. But I can't think what could be.
